src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,36 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/control-has-associated-label */
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
 
-export const TodoList: React.FC = () => {
-  return (
-    <>
-      <p className="notification is-warning">
-        There are no todos matching current filter criteria
-      </p>
```

Comment:
You need to add this element if no todos match filter criteria
<img width="1406" alt="Screenshot 2024-02-14 at 23 00 50" src="https://github.com/mate-academy/react_redux-list-of-todos/assets/26187792/4e9cb0c8-98ca-46bc-bbf3-79408384bc9c">